manywheel/build_libtorch.sh

```diff
@@ -300,4 +336,5 @@ done
 # Copy wheels to host machine for persistence before testing
 if [[ -n "$PYTORCH_FINAL_PACKAGE_DIR" ]]; then
     cp /$LIBTORCH_HOUSE_DIR/libtorch*.zip "$PYTORCH_FINAL_PACKAGE_DIR"
+    cp /$LIBTORCH_HOUSE_DIR/debug-libtorch*.zip "$PYTORCH_FINAL_PACKAGE_DIR"
```

Comment:
TODO: Resolve if this will mess up the binary uploads (upload to s3 should pick up this zip automatically but maybe it should be going into its own folder instead of the one with all the other `libtorch`es?)